pkg/controllers/disruption/events/events.go

```diff
@@ -102,23 +102,26 @@ func Unconsolidatable(node *v1.Node, nodeClaim *v1beta1.NodeClaim, reason string
 
 // Blocked is an event that informs the user that a NodeClaim/Node combination is blocked on deprovisioning
 // due to the state of the NodeClaim/Node or due to some state of the pods that are scheduled to the NodeClaim/Node
-func Blocked(node *v1.Node, nodeClaim *v1beta1.NodeClaim, reason string) []events.Event {
-	return []events.Event{
-		{
+func Blocked(node *v1.Node, nodeClaim *v1beta1.NodeClaim, reason string) (evs []events.Event) {
+	if node != nil {
```

Comment:
Curious why we're reasoning about both types of objects here.

Reply:
We emit an event for both the node and nodeclaim. If there's no node or nodeclaim object associated with the computation we're doing, we won't emit. Is that what you're asking?